src/blosc2/ndarray.py

```diff
@@ -817,7 +896,17 @@ def to_cframe(self):
         See Also
         --------
         :func:`~blosc2.ndarray_from_cframe`
+            This function can be used to reconstruct an NDArray from the serialized bytes.
 
+        Examples
+        --------
+        >>> import blosc2
+        >>> a = blosc2.full(shape=(1000, 1000), fill_value=9, dtype='i4')
+        >>> # Get the bytes object containing the serialized instance
+        >>> cframe_bytes = a.to_cframe()
+        >>> # Verify the type of the returned object
+        >>> isinstance(cframe_bytes, bytes)
+            True
         """
```

Comment:
Instead of this verification, I think it would be more illustrative to use `blosc2.ndarray_from_cframe` and print some interesting info out of it (like the shape and dtype).

Reply:
Done!